kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/job/AbstractPipelineJob.java

```diff
@@ -70,7 +69,7 @@ public Collection<Integer> getShardingItems() {
     protected void addTasksRunner(final int shardingItem, final PipelineTasksRunner tasksRunner) {
         tasksRunnerMap.put(shardingItem, tasksRunner);
         PipelineJobProgressPersistService.addJobProgressPersistContext(getJobId(), shardingItem);
-        distributedBarrier.persistEphemeralChildrenNode(PipelineMetaDataNode.getJobBarrierEnablePath(getJobId()), shardingItem);
+        distributedBarrier.persistEphemeralChildrenNode(getJobId(), shardingItem);
```

Comment:
Looks `persistEphemeralChildrenNode` first parameter is changed to jobId, but it might be `enable` or `disable` path before, is it expected? Or is the impl logic is changed?